packages/multichain-account-service/src/providers/SolAccountProvider.ts

```diff
@@ -11,21 +11,13 @@ import { KeyringClient } from '@metamask/keyring-snap-client';
 import type { SnapId } from '@metamask/snaps-sdk';
 import { HandlerType } from '@metamask/snaps-utils';
 import type { Json, JsonRpcRequest } from '@metamask/utils';
-import type { MultichainAccountServiceMessenger } from 'src/types';
 
-import { SnapAccountProvider } from './SnapAccountProvider';
+import {
+  SnapAccountProvider,
+  type SnapAccountProviderConfig,
+} from './SnapAccountProvider';
 import { withRetry, withTimeout } from './utils';
-
-export type SolAccountProviderConfig = {
```

Comment:
IMO we could keep those as aliases? I think having them matching their actual provider class name is good (especially if one day we start having diverging config again)

So maybe this could just be `export type SolAccountProviderConfig = SnapAccountProviderConfig` for now?